### PR TITLE
[Dotenv] Mention the `overrideExistingVars` parameter of `DotEnv`

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Related to https://github.com/symfony/symfony/pull/52090

I think this is a good place to mention it, as the DotEnv component doesn't have a proper documentation page

### CODE REVIEW

File: configuration.rst [hunk 1]
```diff
@@ -872,6 +872,27 @@ the env files ending in ``.local`` (``.env.local`` and ``.env.<environment>.loca
...
+:method:`Symfony\\Component\\Dotenv\\Dotenv::loadEnv`,
+:method:`Symfony\\Component\\Dotenv\\Dotenv::bootEnv`, and
+:method:`Symfony\\Component\\Dotenv\\Dotenv::populate` methods accept an optional
+``overrideExistingVars`` parameter that allows you to override existing environment
+variables set by the system::
+
+    use Symfony\Component\Dotenv\Dotenv;
+
+    $dotenv = new Dotenv();
+    $dotenv->loadEnv(__DIR__.'/.env', null, 'dev', ['test'], true);
```
Comment (OskarStark) [inline configuration.rst:889]: I would use named arguments here and only add the required one + the overrideExistingVars parameter

**Review by OskarStark — CHANGES_REQUESTED** [review]
See my comment

File: configuration.rst (hunk 1 above)
Comment (alexandre-daubois) [reply configuration.rst:889]: Yes but 5.4 minimal req is PHP 7.2. Shouldn't the example be compatible with the lowest possible PHP version ?

Comment (OskarStark) [reply configuration.rst:889]: Oh indeed, but we can do it after upmerge

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
cc @oleg-andreyev

**Comment by OskarStark** [comment]
Open for a PR to adjust the PHPDoc in the code repo, like proposed by stof?

**Review by oleg-andreyev — APPROVED** [review]

File: configuration.rst (hunk 1 above)
Comment (alexandre-daubois) [reply configuration.rst:889]: I'll provide the PR :slightly_smiling_face:

**Comment by alexandre-daubois** [comment]
Here you go :slightly_smiling_face:

**Comment by javiereguiluz** [comment]
Nice contribution as always. Thanks Alex!

**Pull request merged**